autometa/common/coverage.py

```diff
@@ -170,7 +171,7 @@ def get(
     if from_spades:
         return from_spades_names(records=[rec for rec in SeqIO.parse(fasta, "fasta")])
 
-    try:
+    with tempfile.TemporaryDirectory() as tempdir:
```

Comment:
Do we want to remove files if the run fails? How will the user be able to troubleshoot the error?
```suggestion
    try:
```

Reply:
But isn't putting `finally` at the end of the `try and except` block doing the same thing? I noticed that the `finally` statememt is executed even if an error is raised, so instead `finally` also removing the `tempdir` even if the tool fails?